vowpalwabbit/parse_args.cc

```diff
@@ -441,7 +441,10 @@ input_options parse_source(vw& all, options_i& options)
               .help(
                   "use gzip format whenever possible. If a cache file is being created, this option creates a "
                   "compressed cache file. A mixture of raw-text & compressed inputs are supported with autodetection."))
-      .add(make_option("no_stdin", all.stdin_off).help("do not default to reading from stdin"));
+      .add(make_option("no_stdin", all.stdin_off).help("do not default to reading from stdin"))
+      .add(make_option("enable_json_chain_hash", parsed_options.enable_json_chain_hash)
```

Comment:
[Minor][Nit] Consider using a shorter argument key (it makes it easier to use on the command-line, if needed), for example: "chain_hash_json".